### ISSUE TITLE
AcrylicHost

### ISSUE THREAD

**Issue by delta1186** [cretueusebiu/valet-windows#76]
I started having an issue with valet not resolving my parked sites. After many failed attempts I partially got things working by manually setting the ip6 setting. However I did not need to do this before. My question is is the AcrylicHost.txt file in the .valet folder supposed to be back? The stub in the source is not.

**Comment by cretueusebiu** [comment]
The stub it's still [there](https://github.com/cretueusebiu/valet-windows/blob/master/cli/stubs/AcrylicHosts.txt). 

Make sure the services are stopped (`valet stop` run as admin) then uninstall it (`valet uninstall`) and then reinstall it again. On some systems you may need to configure Acrylic manually, there's a link in the readme.

**Comment by delta1186** [comment]
I understand, but my confusion is that it worked for over two weeks. It noticed it had stopped working yesterday.  

After searching around for clues and stopping and uninstalling valet-windows, I noticed that the AcrylicHost file in my .valet folder was blank (not back :)). When I saw the stub version in code I was wondering if my local .valet copy should have the same text in it.  Should the .valet folder copy mirror the stub folder version?

Currently I’ve manually set the ip6 dns to ::1 which made valet the dns start working for valet. However the side effect was that my local domain dns can no longer resolve. I didn’t set the ip4 dns yet because I am already using a static ip. I guess at worse case I can switch one of the dns addresses to the loopback ip.

**Comment by delta1186** [comment]
Leaving a comment to say I got it working.  I did end up completely removing valet-windows and associated folders, but that did not work.  I manually set the ip4 and 6 dns settings, but that broke my local domain dns.  What worked for me is I just set ip4 dns (127.0.0.1), but just added it as an additional 3rd dns under advanced options since I was already using a static ip.  I did not add the ip6 ::1 dns and left that all set to automatic.  After that my dns started working.
